esp/src/src-react/hooks/grid.tsx

```diff
@@ -138,7 +138,7 @@ function useFluentStoreGrid({
     filename
 }: useFluentStoreGridProps): useFluentStoreGridResponse {
 
-    const constColumns = useConst({ ...columns });
+    const constColumns = useDeepMemo(() => ({ ...columns }), [columns], [columns]);
```

Comment:
Previously `useDeepMemo(() => ({ ...columns }), [], [columns]);` resulted in the state being up to date in the column formatter callbacks (to include the correct scope path in the urls). But after rebasing that stopped updating. Making the second param also [columns] fixed that.

Reply:
`useDeepMemo(() => (columns), [], [columns]);` is technically the preferred version (you may get a false lint warning with that though...)